package.json

```diff
@@ -21,6 +21,7 @@
     "babelify": "^10.0.0",
     "browserify": "^16.2.3",
     "chai": "^4.3.6",
+    "ethereum-waffle": "^4.0.0-alpha.0",
```

Comment:
Its better to install https://www.npmjs.com/package/@ethereum-waffle/chai

Reply:
done